pkg/opentelemetry-mapping-go/otlp/metrics/simple_mapper.go

```diff
@@ -0,0 +1,99 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package metrics
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+)
+
+// SimpleMapper is a cache-free mapper implementation that directly emits
+// metric values without cumulative-to-delta conversion.
+type SimpleMapper struct {
+	cfg    translatorConfig
+	logger *zap.Logger
+}
+
+// NewSimpleMapper creates a new SimpleMapper without a cache.
+// This mapper emits raw values as Gauges instead of computing deltas.
```

Comment:
This sentence is ambiguous: can we specify what values are the ones that we map to gauges?

I guess you meant something like:
```suggestion
// This mapper emits raw values from OTLP cumulative monotonic Sums as Datadog Gauges, instead of computing deltas and reporting them as Datadog Counts.
```